### PR TITLE
style:add landing page

### PR DESCRIPTION
**Pull request by starver20**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
```
Comment (Kevin-Solomon) [inline index.html]: change the title

**Review by Kevin-Solomon — CHANGES_REQUESTED** [review]
Would love to see responsiveness on the landing page

File: style.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+.btn:active,
+.btn:focus {
+  outline: none;
+}
+
+.landing-button {
+  margin-top: 10rem;
+  padding: 2rem;
+  background-color: var(--primary-color);
+  border-radius: 40px;
```
Comment (BISINIGISHIVANI) [inline style.css]: for border-radius try to give % or rem and now also its fine.

File: index.html (hunk 1 above)
Comment (starver20) [reply index.html]: Thanks, i will make it responsive

File: style.css (hunk 2 above)
Comment (starver20) [reply style.css]: Allright ill change it to rems

**Review by kart-c — APPROVED** [review]
As Kevin suggested you should change the document title. Other than that everything else looks good

**Review by kart-c — APPROVED** [review]
Good work on the responsiveness 👍

**Review by Kevin-Solomon — APPROVED** [review]

File: style.css (hunk 2 above)
Comment (BISINIGISHIVANI) [reply style.css]: 👍 looks good

**Pull request merged**